### PR TITLE
zigbee: Add an additional zboss osif api implementation

### PR DESCRIPTION
**Pull request by tomchy**

### CODE REVIEW

File: subsys/zigbee/osif/zb_nrf_platform.c [hunk 1]
```diff
@@ -566,3 +571,43 @@ void zigbee_enable(void)
...
+	} else if (reas & NRF_RESET_RESETREAS_SREQ_MASK) {
+		return ZB_RESET_SRC_SW_RESET;
+	} else if (reas) {
+		return ZB_RESET_SRC_OTHER;
+	} else {
+		return ZB_RESET_SRC_POWER_ON;
+	}
+
+#endif
+}
```
Comment (rlubos) [inline subsys/zigbee/osif/zb_nrf_platform.c:613]: nit: Missing new line.

**Review by rlubos — COMMENTED** [review]
Looks good, will approve when SHA gets updated.

File: subsys/zigbee/osif/zb_nrf_platform.c (hunk 1 above)
Comment (tomchy) [reply subsys/zigbee/osif/zb_nrf_platform.c]: fixed

**Review by sebastiandraus — APPROVED** [review]

File: west.yml [hunk 2]
```diff
@@ -103,7 +103,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 27ce80161f50254dc07caeb1118a7cbe6b121ce8
+      revision: pull/362/head
```
Comment (rlubos) [inline west.yml:106]: Please update SHA: `e6e02cb83d238fae2f54f084858bd5e49a31afa1`

Comment (tomchy) [reply west.yml]: Updated

**Review by rlubos — APPROVED** [review]

**Comment by joerchan** [comment]
Conflict on west.yml from : https://github.com/nrfconnect/sdk-nrf/pull/3659

Manifest SHA should be the same though. We updated to one before.

**Comment by tomchy** [comment]
@joerchan : Unfortunately not, the e6e02cb is newer than  ecf5d56 :slightly_smiling_face:

**Comment by joerchan** [comment]
> @joerchan : Unfortunately not, the e6e02cb is newer than ecf5d56 🙂

@tomchy That is what I was trying to say, you should have the one you had, since we updated to an older one :)

**Pull request merged**